ros/angel_system_nodes/angel_system_nodes/task_monitor.py

```diff
@@ -98,11 +182,34 @@ def __init__(self):
         self._timer_lock = threading.RLock()
 
         self._activity_action_dict = {
```

Comment:
Is this the mapping that links activity detector output to the task transition functions? If so, we should leave a little comment to that effect.